src/test/ui/method-call-err-msg.stderr

```diff
@@ -0,0 +1,48 @@
+error[E0061]: this function takes 0 parameters but 1 parameter was supplied
+  --> $DIR/method-call-err-msg.rs:25:12
+   |
+15 |     fn zero(self) -> Foo { self }
+   |     ----------------------------- defined here
+...
+25 |     x.zero(0)   //~ ERROR this function takes 0 parameters but 1 parameter was supplied
+   |            ^ expected 0 parameters
+
+error[E0061]: this function takes 1 parameter but 0 parameters were supplied
+  --> $DIR/method-call-err-msg.rs:25:5
+   |
+17 |       fn one(self, _: isize) -> Foo { self }
+   |       -------------------------------------- defined here
+...
+25 | /     x.zero(0)   //~ ERROR this function takes 0 parameters but 1 parameter was supplied
+26 | |      //~^ NOTE expected 0 parameters
+27 | |      //~^ ERROR this function takes 1 parameter but 0 parameters were supplied
+28 | |      //~^ NOTE expected 1 parameter
+29 | |      .one()
```

Comment:
This is a definite regression. It used to produce

```
error[E0061]: this function takes 1 parameter but 0 parameters were supplied
  --> src/main.rs:15:7
   |
5  |      fn one(self, _: isize) -> Foo { self }
   |      -------------------------------------- defined here
...
15 |      .one()     //~ ERROR this function takes 1 parameter but 0 parameters were supplied
   |       ^^^ expected 1 parameter
```

And we really want to keep the old report